tests/scripts/thread-cert/Cert_6_1_05_REEDAttachConnectivity.py

```diff
@@ -80,21 +87,60 @@ def test(self):
         self.simulator.go(5)
         self.assertEqual(self.nodes[ROUTER1].get_state(), 'router')
 
-        self.nodes[REED0].start()
-        self.simulator.go(5)
-        self.assertEqual(self.nodes[REED0].get_state(), 'child')
-
         self.nodes[REED1].start()
         self.simulator.go(5)
         self.assertEqual(self.nodes[REED1].get_state(), 'child')
 
-        self.simulator.go(10)
+        self.nodes[REED2].start()
+        self.simulator.go(5)
+        self.assertEqual(self.nodes[REED2].get_state(), 'child')
 
         self.nodes[ED].start()
         self.simulator.go(10)
         self.assertEqual(self.nodes[ED].get_state(), 'child')
         self.assertEqual(self.nodes[REED1].get_state(), 'router')
 
+        self.collect_ipaddrs()
+        addrs = self.nodes[ED].get_addrs()
+        for addr in addrs:
+            self.assertTrue(self.nodes[REED1].ping(addr))
+
+    def verify(self, pv):
+        pkts = pv.pkts
+        pv.summary.show()
+
+        REED_1 = pv.vars['REED_1']
+        ED = pv.vars['ED']
+        _reed1_pkts = pkts.filter_wpan_src64(REED_1)
+        _ed_pkts = pkts.filter_wpan_src64(ED)
+
+        # Step 2: The DUT MUST send a MLE Parent Request to the
+        # All-Routers multicast address
```

Comment:
Are we validating the all-routers multicast address?

Reply:
yes, we should. Added in the commit 6